x11/src/lib.rs

```diff
@@ -396,8 +396,8 @@ impl Worker {
                             &data,
                         )
                         .expect("Change property");
-                    } else if value.len() < max_length - 24 {
-                        let _ = self.context.connection.change_property8(
+                    } else if value.len() < max_length {
+                        let _ = self.context.connection.change_property32(
```

Comment:
So changing this all to `u32` I was thinking this could be improved, is this correct?

Reply:
So apparently this doesn't work.